mwdb/model/file.py

```diff
@@ -100,13 +101,14 @@ def get_or_create(cls, file, parent=None, metakeys=None, share_with=None):
                 ).put_object(
                     app_config.mwdb.s3_storage_bucket_name,
                     file_obj._calculate_path(),
-                    file.stream,
+                    file_stream,
                     file_size
                 )
             else:
-                file.save(file_obj._calculate_path())
+                with open(file_obj._calculate_path(), "wb") as f:
+                    copyfileobj(file_stream, f)
```

Comment:
`shutil` is already imported.

```suggestion
                    shutil.copyfileobj(file_stream, f)
```